website/docs/04 - Model/06 - Querying/02 - Query Operators.md

```diff
@@ -154,7 +154,7 @@ The query which will be executed on the MultiValue database is:
 select ITEM with DESCRIPTION like "...Bed..."
 ```
 
-:::caution
+:::warning[caution]
```

Comment:
Similar note here -- can just be `:::warning`

Reply:
Fixed.